frameworks/ruby/rails6/ActiveRecordMigrations.sh

```diff
@@ -0,0 +1,998 @@
+# Rails Guide on Active Records
+# https://guides.rubyonrails.org/active_record_migrations.html
+
+source helper.sh
+
+# 1. Migration Overview
+# check_create_products_migration checks that the create product migration works correctly
+function check_create_products_migration(){
+  # create the migration file and write the content
+  rails_generate_migration_with_content "CreateProduct1s" "class CreateProduct1s < ActiveRecord::Migration[6.1]
+  def change
```

Comment:
There is inconsistent indentation in this file, everything should be indented with two spaces.

Reply:
This happened while pasting rails code from guides and I didn't notice :(

Reply:
Fixed